Sources/Widgets/Widgets3D/ShapeWidget/behavior.js

```diff
@@ -371,29 +371,26 @@ export default function widgetBehavior(publicAPI, model) {
     ) {
       return macro.VOID;
     }
-
+    const normal = model.camera.getDirectionOfProjection();
     if (!model.point2) {
-      // Update orientation to match the camera's plane
-      // if the corners are not yet placed
-      const normal = model.camera.getDirectionOfProjection();
       const up = model.camera.getViewUp();
       const right = [];
+      normal[0] *= -1;
+      normal[1] *= -1;
+      normal[2] *= -1;
       vec3.cross(right, up, normal);
```

Comment:
why?

Reply:
My bad on this one, these lines were for test purposes and are now removed